ProjectSettings/TagManager.asset

```diff
@@ -36,6 +36,9 @@ TagManager:
   - LAMP
```

Comment:
What are tags?

Reply:
Tags can be attached to gameObjects to differentiate them from others - we use them in sensorManager to distinguish the subscriber toggles from any other gameobject in the scene.